### PR TITLE
Add tooltip for showing image data

### PR DESCRIPTION
**Pull request by KaroMourad**
- Add tooltip to image explore.
- Open tooltip with image data inside on hovering on the image and close it on mouse leave.
- Add tooltip data controller in the controls section for tooltip content changing.
- Add hide/show switcher in tooltip control popover.

### CODE REVIEW

**Comment by KaroMourad** [comment]
Resolves [#1024](https://github.com/aimhubio/aim/issues/1024)

**Review by rubenaprikyan — APPROVED** [review]
lgtm!

File: aim/web/ui/src/components/ChartPanel/ChartPanel.scss [hunk 1]
```diff
@@ -2,38 +2,43 @@
...
+      overflow: auto;
+      margin: 0;
+      &__chartBox {
+        min-height: 50%;
+        padding: 4px;
+        box-shadow: -1px -1px 0 0 #e8e8e8; // var(--grey-lighter);
+      }
+    }
+    &__controls {
+      width: 3.75rem;
```
[the remark this answers is not in the document]
Comment (KaroMourad) [reply aim/web/ui/src/components/ChartPanel/ChartPanel.scss:35]: thanks

File: aim/web/ui/src/components/ImagesSet/ImagesSet.d.ts [hunk 2]
```diff
@@ -1,3 +1,5 @@
+import { IFocusedState } from '../../types/services/models/metrics/metricsAppModel';
```
Comment (VkoHov) [inline aim/web/ui/src/components/ImagesSet/ImagesSet.d.ts:1]: Please change it to import { IFocusedState } from 'types/services/models/metrics/metricsAppModel';

**Review by VkoHov — APPROVED** [review]

File: aim/web/ui/src/components/ImagesSet/ImagesSet.d.ts (hunk 2 above)
Comment (KaroMourad) [reply aim/web/ui/src/components/ImagesSet/ImagesSet.d.ts:1]: already changed thanks

**Pull request merged**